Tutorials/CustomHamiltonian.ipynb

```diff
@@ -104,15 +104,15 @@
     "Now, we need to create the relevant bond operators. Since we have two type of terms in our Hamiltonian, we need two different bond operators. To do we simply express the iteraction term as a matrix in the computational basis of the relevant local Hilbert space. In our case, since we are dealing with spin-half degrees of freedom, the local basis of bond is simply $\\lvert \\uparrow \\uparrow \\rangle$, $\\lvert \\uparrow \\downarrow \\rangle$, $\\lvert \\downarrow \\uparrow \\rangle$, $\\lvert \\downarrow \\downarrow \\rangle$. Since\n",
```

Comment:
Some small typos left: 
to do -> to do this (I guess?)
iteraction -> interaction

Reply:
OK done in the commit